src/App.js

```diff
@@ -1,24 +1,41 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import smilies from './assets/smilies.png';
+import i6 from './assets/i6.png';
+import social from './assets/social.png';
+import play from './assets/play.png';
+import appstorebtn from './assets/appstore_button.png';
+import menu_icon from './assets/icons/menu_icon.png';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div class='main-div'>
+        <div class='left-container'>
+          <div class='menu'>
+            <img src={menu_icon} alt='menu'></img>
```

Comment:
If you aren't planning to put something inside the tag you could just do it one self close tag, like this... 
```suggestion
            <img src={menu_icon} alt='menu' />
```